.github/CODEOWNERS

```diff
@@ -131,13 +131,13 @@ Tasks/Common/VstsAzureRestHelpers_/  @Ajay-MS  @rajatagrawal-dev
 
 Tasks/Common/azure-arm-rest/   @Ajay-MS    @bishal-pdmsft
 
-Tasks/Common/azure-arm-rest-v2/   @nadesu    @chshrikh     @bishal-pdmsft
+Tasks/Common/azure-arm-rest-v2/   @nadesu @v-jkarri   @chshrikh     @bishal-pdmsft
 
 Tasks/Common/azure-blobstorage-artifactProvider/   @omeshp   @hiyadav    @bishal-pdmsft
 
 Tasks/Common/codeanalysis-common/    @microsoft/akvelon-build-task-team
 
-Tasks/Common/coveragepublisher/     @nadesu   @ganesp
+Tasks/Common/coveragepublisher/     @nadesu @v-jkarri  @ganesp
```

Comment:
'coveragepublisher' is not own by environment, you can remove your name from here

Reply:
Removed my name.